common/preprocess.ts

```diff
@@ -101,15 +128,35 @@ const getStatusData = (
           : label || unknownLabelText
       );
       progress.colors.push(theme.graphColors[colors.get(identifier) ?? color]);
-      if (sentiment == Sentiment.Positive) {
-        progress.good = progress.good + statusCount;
+      if (sentiment == Sentiment.Positive || identifier == 'NOT_STARTED') {
```

Comment:
Nitpick: It's a good practice to use strict equality in JS (`===`). The only exception being checking for `null` or `undefined` where `something != null` comes in handy.